backend/src/main/java/gov/cdc/usds/simplereport/api/organization/OrganizationDataResolver.java

```diff
@@ -0,0 +1,51 @@
+package gov.cdc.usds.simplereport.api.organization;
+
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import gov.cdc.usds.simplereport.api.model.ApiFacility;
+import gov.cdc.usds.simplereport.api.model.ApiOrganization;
+import gov.cdc.usds.simplereport.api.model.ApiProvider;
+import gov.cdc.usds.simplereport.db.model.DeviceType;
+import gov.cdc.usds.simplereport.db.model.Facility;
+import gov.cdc.usds.simplereport.db.model.Organization;
+import gov.cdc.usds.simplereport.service.OrganizationService;
+import graphql.kickstart.tools.GraphQLResolver;
+
+@Service
+public class OrganizationDataResolver implements GraphQLResolver<ApiOrganization> {
+
+	private static final Logger LOG = LoggerFactory.getLogger(OrganizationDataResolver.class);
+
+	@Autowired
+	private OrganizationService _orgService;
+
+	public ApiFacility getTestingFacility(ApiOrganization o) {
+		LOG.error("Asked me for a facility for {}", o);
```

Comment:
Why are we logging these as errors?

Reply:
Because we forgot to take out the debug code from when we were making sure that we understood how `GraphQLResolver<?>` beans worked. 😳